src/app/components/Feedback/Feedback.jsx

```diff
@@ -1,217 +1,48 @@
-                      Comments*
-                    </Label>
-                    <textarea
-                      id="feedback-textarea-comment"
-                      name="feedback"
-                      value={fields.feedback}
-                      ref={this.commentText}
-                      rows="8"
-                      aria-required="true"
-                      tabIndex="0"
-                      onChange={this.handleInputChange}
-                    />
-                    <HelperErrorText
-                      id="helper-text"
-                      isInvalid={commentInputError}
-                      text={commentInputError ? 'Please fill out this field' : ''}
-                    />
-                  </div>
-                  <div>
-                    <TextInput
-                      id="feedback-input-email"
-                      labelText={
-                        <>Email <span>(required if you would like a response from us)</span></>
-                      }
-                      name='email'
-                      onChange={this.handleInputChange}
-                      ref={this.emailInput}
-                      // isRequired
-                      type="email"
-                      value={fields.email}
-                      marginTop="l"
-                    />
-                  </div>
-                  <div className="privacy-policy">
-                    <Link
-                      href="https://www.nypl.org/help/about-nypl/legal-notices/privacy-policy"
-                      target="_blank"
-                    >
-                      Privacy Policy
-                    </Link>
-                  </div>
-                  <Button
-                    aria-expanded={!showForm}
-                    aria-controls='feedback-menu'
-                    buttonType="secondary"
-                    id="cancel-feedback"
-                    className={`cancel-button ${!showForm ? 'hidden' : ''}`}
-                    onClick={e => this.deactivateForm(e)}
-                    type="reset"
-                  >
-                    Cancel
-                  </Button>
-
-                  <Button
-                    className="submit-button"
-                    id="submit-feedback"
-                    type="submit"
-                  >
-                    Submit
-                  </Button>
-                </form>
-              </>
-            )}
-            {success && (
-              <p>
-                Thank you for submitting your comments.
-                If you requested a response, our service staff
-                will get back to you as soon as possible.
-              </p>
-            )}
-          </div>
-        </FocusTrap>
-      </div>
-    );
-  }
+  return <>
+    <FeedbackBox
+      onSubmit={submitFeedback}
+      isOpen={isOpen}
+      onClose={closeAndResetItemMetadata}
+      onOpen={onOpen}
+      descriptionText='We are here to help!'
```

Comment:
Just curious, is this text in the Figma design?

Reply:
there werent designs for this. it was the text that was already in the original feedback box

Reply:
Makes sense to keep it.